src/App.js

```diff
@@ -14,6 +14,11 @@ class App extends Component {
       redditData: [],
       twitterData: [],
       hackerNoonData: [],
+      sourceToggle: {
```

Comment:
Could you enable all the sources toggle buttons by default?
Currently, the 3 sources are enabled (the data is displayed) but the buttons are disabled.